plugins/inputs/burrow/burrow_test.go

```diff
@@ -27,6 +27,7 @@ func getResponseJSON(requestURI string) ([]byte, int) {
 	}
 
 	// respond with file
+	//nolint:errcheck // should fail tests later on if this read fails
 	b, _ := os.ReadFile(jsonFile)
 	return b, code
```

Comment:
Since this function was called nested a few functions deep in test cases with no access to `testing.T`, I decided to panic here if there is an error instead